src/data/Dhis2Import.ts

```diff
@@ -1,106 +1,124 @@
 import i18n from "../locales";
+import { D2Api } from "@eyeseetea/d2-api/2.33";
+import { Ref } from "../domain/entities/ReferenceObject";
 
-type Status = "OK" | "ERROR";
+export type Status = "OK" | "ERROR";
 
-export interface ImportPostResponse {
+type ImportStats = {
+    created: number;
+    deleted: number;
+    ignored: number;
+    updated: number;
+    total: number;
+};
+
+export type ImportPostResponse = {
+    status: Status;
+    response: Ref;
+};
+
+export type ImportReportResponse = {
     status: Status;
-    message?: string;
-    response?: {
-        status: SynchronizationStatus;
-        imported: number;
-        updated: number;
-        deleted: number;
-        ignored: number;
-        total: number;
-        importSummaries?: Array<{
-            responseType: "ImportSummary";
-            description?: string;
-            status: SynchronizationStatus;
-            href?: string;
-            importCount: {
-                imported: number;
-                updated: number;
-                deleted: number;
-                ignored: number;
+    validationReport: {
+        errorReports: [
+            {
+                uid: string;
+                message: string;
+            }
+        ];
+    };
+    stats: ImportStats;
+    bundleReport?: {
+        status: Status;
+        stats: ImportStats;
+        typeReportMap: {
+            [type: string]: {
+                trackerType: string;
+                stats: ImportStats;
+                objectReports: {
+                    uid: string;
+                };
             };
-            reference?: string;
-            conflicts?: {
-                object: string;
-                value: string;
-            }[];
-            // Only for TEI import
-            enrollments?: ImportPostResponse["response"];
-        }>;
+        };
     };
-}
+};
 
 export function processImportResponse(options: {
     title: string;
     model: string;
-    importResult: ImportPostResponse;
+    importResult: ImportReportResponse;
     splitStatsList: boolean;
 }): SynchronizationResult {
     const { title, model, importResult, splitStatsList } = options;
-    const { message, response } = importResult;
-    const status = response ? response.status : "ERROR";
-
-    if (!response) return { title, status, message, rawResponse: importResult };
-
-    // Add inner import summaries
-    const importSummaries = _(response.importSummaries)
-        .flatMap(importSummary => [importSummary, ...(importSummary.enrollments?.importSummaries || [])])
-        .value();
 
-    const aggregatedStatus =
-        _(importSummaries)
-            .map(summary => summary.status)
-            .find(status => status !== "SUCCESS") || status;
+    const { bundleReport, status, validationReport } = importResult;
+    const message = status === "OK" ? "Import was successful" : "Import failed";
```

Comment:
If we need translations for this it should be: `i18n.t("Import was successful")` and `i18n.t("Import failed")`